### PR TITLE
fix: allow the member of the org to leave

### PR DESCRIPTION
**Pull request by JivusAyrus**
- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

**Review by StarpTech — APPROVED** [review]
LGTM

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 1]
```diff
@@ -2699,6 +2700,17 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           organizationID: authContext.organizationId,
         });
 
+        await auditLogRepo.addAuditLog({
+          organizationId: authContext.organizationId,
+          auditAction: 'organization_member.left',
```
Comment (StarpTech) [inline controlplane/src/core/bufservices/PlatformService.ts:2705]: It would be `organization.left`

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 2]
```diff
@@ -2699,6 +2700,17 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
...
+          action: 'left',
+          actorId: authContext.userId,
+          auditableType: 'organization',
+          auditableDisplayName: org.name,
+          actorDisplayName: authContext.userDisplayName,
+          actorType: 'user',
```
Comment (StarpTech) [inline controlplane/src/core/bufservices/PlatformService.ts:2711]: Please use ` actorType: authContext.auth === 'api_key' ? 'api_key' : 'user',`

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 3]
```diff
@@ -2088,8 +2088,8 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
 
         await auditLogRepo.addAuditLog({
           organizationId: authContext.organizationId,
-          auditAction: 'organization_member.deleted',
-          action: 'deleted',
+          auditAction: 'organization_member.removed',
+          action: 'removed',
```
Comment (StarpTech) [inline controlplane/src/core/bufservices/PlatformService.ts:2092]: Why did you rename it?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: controlplane/src/core/bufservices/PlatformService.ts (hunk 2 above)
Comment (JivusAyrus) [reply controlplane/src/core/bufservices/PlatformService.ts:2711]: Done

File: controlplane/src/core/bufservices/PlatformService.ts (hunk 1 above)
Comment (JivusAyrus) [reply controlplane/src/core/bufservices/PlatformService.ts]: Done

File: controlplane/src/core/bufservices/PlatformService.ts (hunk 3 above)
Comment (StarpTech) [reply controlplane/src/core/bufservices/PlatformService.ts:2092]: I see your point but we deleted the membership not the user.

**Review by StarpTech — APPROVED** [review]
Lgtm
